File: Ayush-Gupta/905. Sort Array By Parity.cpp
class Solution {
public:
    vector<int> sortArrayByParity(vector<int>& nums) {
        for(int i=0, j=0;i<nums.size();i++) {
            if(nums[i]%2==0) {
                swap(nums[i], nums[j++]);
            }
        }
        return nums;
    }
};

File: Namrata-Sanger/905. Sort Array By Parity.js
/*
Day 19:
905. Sort Array By Parity
Link: https://leetcode.com/problems/sort-array-by-parity/
Level: Easy
*/

var sortArrayByParity = function(nums) {
    return nums.sort((a,b)=>(a%2) - (b%2));
};
